prost-build/src/code_generator.rs

```diff
@@ -87,14 +79,8 @@ impl OneofField {
     }
 }
 
-impl CodeGenerator<'_> {
-    pub fn generate(
-        config: &mut Config,
-        message_graph: &MessageGraph,
-        extern_paths: &ExternPaths,
-        file: FileDescriptorProto,
-        buf: &mut String,
-    ) {
+impl<'a, 'b> CodeGenerator<'a, 'b> {
+    pub fn generate(context: &mut Context<'b>, file: FileDescriptorProto, buf: &mut String) {
```

Comment:
This is an API breaking change, right? I don't understand how the `Context` instance should be created by `prost` users as there is no documentation changes included.

Reply:
Yes `pub(crate)` is a great idea